src/specs/settings.tsx

```diff
@@ -46,16 +48,24 @@ export interface GroupBrushExtent {
   extent: [number, number];
 }
 export interface XYBrushArea {
+  chartType: typeof ChartTypes.XYAxis;
   x?: [number, number];
   y?: Array<GroupBrushExtent>;
 }
 
 export type XYChartElementEvent = [GeometryValue, XYChartSeriesIdentifier];
 export type PartitionElementEvent = [Array<LayerValue>, SeriesIdentifier];
+export type HeatmapElementEvent = [Cell, SeriesIdentifier];
 
-export type ElementClickListener = (elements: Array<XYChartElementEvent | PartitionElementEvent>) => void;
-export type ElementOverListener = (elements: Array<XYChartElementEvent | PartitionElementEvent>) => void;
-export type BrushEndListener = (brushArea: XYBrushArea) => void;
+export type HeatmapBrushEvent = { cells: Cell[]; x: any[]; y: any[]; chartType: typeof ChartTypes.Heatmap };
```

Comment:
Can we tighten the type here?

```suggestion
export type HeatmapBrushEvent = { cells: Cell[]; x: (string | number)[]; y: (string | number)[]; chartType: typeof ChartTypes.Heatmap };
```

Reply:
Tried to commit suggestion and GitHub immediately resolved the conversation. Fixed in https://github.com/elastic/elastic-charts/pull/831/commits/c22ffc8020d11e2005e6c18715c575c0c11d2d7f